lib/matplotlib/contour.py

```diff
@@ -1731,10 +1743,10 @@ def _initialize_x_y(self, z):
 
         An existing `.QuadContourSet` does not get notified if
         properties of its colormap are changed. Therefore, an explicit
-        call `.QuadContourSet.changed()` is needed after modifying the
+        call `~.ContourSet.changed()` is needed after modifying the
         colormap. The explicit call can be left out, if a colorbar is
         assigned to the `.QuadContourSet` because it internally calls
-        `.QuadContourSet.changed()`.
+        `~.ContourSet.changed()`.
```

Comment:
There is an exception for `QuadContourSet.changed()` in `missing-references.json` that can be deleted due to this change?

Reply:
yes.  It may also make sense to drop this change in this PR and just update the exceptions.

I also went down a rabbit hole of sorting out how we generate these tables and why the other failures happen.